src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -18,84 +19,93 @@ namespace NUnit.Engine.Services
     /// but only one, ProcessAgent is implemented
     /// at this time.
     /// </summary>
-    public partial class TestAgency : ServerBase, ITestAgency, IService
```

Comment:
Should `ServerBase.cs` be removed in this PR? Looks like it's now ported over to `TestAgencyRemotingTransport`?